tests/test_candidate.py

```diff
@@ -79,6 +79,14 @@ def test_dmtime_snr_opt_snr(cand):
     assert pytest.approx(cand.optimize_dm()[0], rel=2) == 475
 
 
+def test_h5(cand):
+    cand.get_chunk()
+    cand.dedisperse()
+    cand.dmtime()
+    cand.save_h5()
+    assert os.path.isfile(str(cand.id) + ".h5")
```

Comment:
It would be better to delete the file after the `assert` statement, unless that same h5 file is being used for another test.